.gitignore

```diff
@@ -7,3 +7,6 @@ yarn.lock
 vite.config.js.timestamp-*
 /packages/create-svelte/template/CHANGELOG.md
 .test-tmp
+.test-output
+
+*storybook.log
```

Comment:
nit: it'd be nice to add a newline to the end of this file and the `.gitignore` in the community template

Reply:
huh, looks like storybook added that. should be fixed now 👍